share/spice/search_code/search_code.js

```diff
@@ -1,52 +1,62 @@
-function ddg_spice_search_code(api_response) {
+(function(env) {
     "use strict";
+    env.ddg_spice_search_code = function(api_result) {
 
-    var query = api_response.query;
-    var data = api_response.results;
+        var query = api_result.query;
+        var data = api_result.results;
 
-    if(!data.length || data.length === 0) {
-        return Spice.failed('search_code');
-    }
-
-    var result;
-    for (var i = 0; i < data.length; i++) {
-        var checkRelevancy = [data[i].name, data[i].displayname, data[i].namespace].join(" ");
-        if (DDG.isRelevant(checkRelevancy, [], 2)) {
-            result = data[i];
-            break;
+        if(!data.length || data.length === 0) {
+            return Spice.failed('search_code');
         }
-    }
 
-    if (!result) {
-        return Spice.failed('search_code');
-    }
+        var result;
+        for (var i = 0; i < data.length; i++) {
+            var checkRelevancy = [data[i].name, data[i].displayname, data[i].namespace].join(" ");
+            if (DDG.isRelevant(checkRelevancy, [], 2)) {
+                result = data[i];
+                break;
+            }
+        }
 
-    function formatName(result) {
-        var formatted_name = result.name;
+        if (!result) {
+            return Spice.failed('search_code');
+        }
 
-        if (result.displayname !== '' || result.namespace !== '') {
-            formatted_name += ' (';
-            if (result.displayname !== '') {
-                formatted_name += result.displayname;
-            }
-            if (result.namespace !== '') {
-                formatted_name += (result.displayname ? ', ' : '') + result.namespace;
+        function formatName(result) {
```

Comment:
We can probably get rid of building this string an use a `subtitle` to display any extra relevant information